src/analysis/inner_product_space/symmetric.lean

```diff
@@ -154,4 +154,39 @@ end
 
 end complex
 
+/-- Polarization identity for symmetric linear maps.
+See `inner_map_polarization` for the complex version without the symmetric assumption. -/
+lemma is_symmetric.inner_map_polarization {T : E →ₗ[𝕜] E} (hT : T.is_symmetric) (x y : E) :
+  ⟪T x, y⟫ = (⟪T (x + y), x + y⟫ - ⟪T (x - y), x - y⟫ -
+    I * ⟪T (x + (I : 𝕜) • y), x + (I : 𝕜) • y⟫ +
+    I * ⟪T (x - (I : 𝕜) • y), x - (I : 𝕜) • y⟫) / 4 :=
+begin
+  rcases @I_mul_I_ax 𝕜 _ with (h | h),
+  { simp_rw [h, zero_mul, sub_zero, add_zero, map_add, map_sub, inner_add_left,
+             inner_add_right, inner_sub_left, inner_sub_right, hT x _,
+             ← inner_conj_symm x (T y)],
```

Comment:
Better style is to indent two spaces. That avoids having different indentations depending on the name of the tactic.
```suggestion
  { simp_rw [h, zero_mul, sub_zero, add_zero, map_add, map_sub, inner_add_left,
      inner_add_right, inner_sub_left, inner_sub_right, hT x _, ← inner_conj_symm x (T y)],
```